### PR TITLE
dist: add community project - uptime robot dashboard

### PR DESCRIPTION
**Pull request by matthiasbaldi**

### CODE REVIEW

**Comment by chrisys** [comment]
Thanks for the contribution! It doesn't look like this will build correctly as you haven't added the definition for the repository link or the username (`secanis`) in the lower part of the file.

**Comment by chrisys** [comment]
Additionally your commit message should include the `Change-type` as per our contribution guidelines here: https://github.com/balena-io/docs/blob/master/CONTRIBUTING.md - without this the CI will fail to build and will not allow a merge.

File: pages/learn/more/examples/projects.md [hunk 1]
```diff
@@ -195,6 +195,12 @@ A simple tool for reading temperatures from a USB-enabled thermometer. This
 project is used as the backend to [efwe][efwe]'s awesome temperature
 visualisation at [123k.de](http://123k.de).
 
+### Uptime Robot Dashboard
+[Repository][uptime-robot-dashboard] by [secanis][secanis]
+
+A dashboard which is visualizing the [Uptime Robot API](https://uptimerobot.com/) on a Raspberry PI, deployed over the Balena Cloud.
```
Comment (chrisys) [inline pages/learn/more/examples/projects.md]:
```suggestion
A dashboard which is visualizing the [Uptime Robot API](https://uptimerobot.com/) on a Raspberry Pi, deployed via balenaCloud.
```

File: pages/learn/more/examples/projects.md [hunk 2]
```diff
@@ -195,6 +195,12 @@ A simple tool for reading temperatures from a USB-enabled thermometer. This
...
+This dashboard could be used on TV's or on monitors.
```
Comment (chrisys) [inline pages/learn/more/examples/projects.md]:
```suggestion
This dashboard can be used on TVs or on monitors.
```

**Comment by chrisys** [comment]
@balena-ci retest

**Comment by matthiasbaldi** [comment]
Do I have to fix something else? :)

**Comment by chrisys** [comment]
@matthiasbaldi I'm just trying to figure out why the CI fails, it looks like it might be because `Change-type: patch` is the title of the commit message rather than in the body.

**Comment by matthiasbaldi** [comment]
Ah sorry, damit... I ment I have to write this stuff into the title and the rest of the commit message into the body xD

**Comment by chrisys** [comment]
@matthiasbaldi did you want to update your commit messages again or if not let me know and I'll close this and submit your changes myself.

**Comment by chrisys** [comment]
@balena-ci retest

**Pull request merged**